Add openvr dependency for RE of BSOpenVR

Nightfallstorm opened the pull request:
OpenVR submodule: https://github.com/ValveSoftware/openvr

alandtse commented:
To be consistent with other mods, can you make `openvr` be in the directory `extern/openvr`? I think that's the standard structure for external dependencies in c++.

Nightfallstorm committed:
Add dependency openvr

This dependency is needed for the RE of BSOpenVR

Additionally, this dependency is rolled back to 1.0.10 to match the version Skyrim VR uses

Nightfallstorm commented:
Moved
